src/contracts/fund/hub/transactions/setupFund.ts

```diff
@@ -0,0 +1,84 @@
+import * as R from 'ramda';
```

Comment:
I think this is a test-only file? Can you please move it to `src/tests/utils`? There are already other files like this.

Reply:
no its not a test file ; it allows to create a fund without investing in it

Reply:
Can be used by bots / Midas as a way to easily spin up a fund

Reply:
True.